sycl/include/CL/sycl/INTEL/esimd/esimd_memory.hpp

```diff
@@ -643,6 +643,198 @@ media_block_store(AccessorTy acc, unsigned x, unsigned y, simd<T, m * n> vals) {
 SYCL_EXTERNAL void slm_init(uint32_t size) {}
 
 #endif
+
+template <typename AccessorTy>
+ESIMD_INLINE ESIMD_NODEBUG uint32_t esimd_get_value(AccessorTy acc) {
+#if defined(__SYCL_DEVICE_ONLY__) && defined(__SYCL_EXPLICIT_SIMD__)
+  return __esimd_get_value(AccessorPrivateProxy::getNativeImageObj(acc));
+#else
+  return __esimd_get_value(acc);
+#endif // __SYCL_DEVICE_ONLY__ && __SYCL_EXPLICIT_SIMD__
+}
+
+/// \brief Raw sends load.
+///
+/// @msgDst the destination operand of send message.
+///
```

Comment:
it is not clear how this parameter is to be used - it is passed by value, yet it is a destination.

Reply:
It's the old value of the destination operand - updated comment to clarify this.